nvflare/private/fed/utils/app_deployer.py

```diff
@@ -83,4 +83,5 @@ def deploy(self) -> str:
                 return "not authorized"
 
         except BaseException as ex:
-            return f"exception {ex} when deploying app {self.app_name}"
+            raise Exception(f"exception {ex} when deploying app {self.app_name}")
+            # return f"exception {ex} when deploying app {self.app_name}"
```

Comment:
the app_deployer was originally just returning "string", but this PR change to raise Exception.

will this cause any issues?